CHANGELOG.md

```diff
@@ -1,18 +1,34 @@
 Changes by Version
 ==================
 
-1.15.0 (unreleased)
+1.15.0 (2019-11-06)
 ------------------
-
 #### Backend Changes
 
 ##### Breaking Changes
```

Comment:
weren't there some changes the the CLI flags? We usually list them (e.g. some flags may have been deprecated and replaced). You could dump `help` for each binary and do a diff (we may want to write & commit a script for that).

Reply:
There was a commit to add firehose flag for tracegen, do we want to add it to the changelog?